examples/vicuna-llm/README.md

```diff
@@ -0,0 +1,57 @@
+# Vicuna: An LLM Chatbot Impressing GPT-4 with 90% ChatGPT Quality
+
+<img src="https://vicuna.lmsys.org/favicon.jpeg" width=256 alt="Vicuna LLM"/>
```

Comment:
Should we use percentage width instead to make it looks better on different screens?

Reply:
Done, set to 25%.